torch_patches/.torch_pin

```diff
@@ -0,0 +1 @@
+#48570
```

Comment:
I think you need a new line but I could be wrong.

Reply:
No new line is OK. But I added a new line in the latest version.